asab/library/contextvars.py

```diff
@@ -0,0 +1,5 @@
+# contextvars.py
+from contextvars import ContextVar
+
+# Define a context variable for tenant
+tenant_var = ContextVar('tenant', default=None)
```

Comment:
Move this into Zookeeper provider.
There has to be an official and documented way how an user can set it.